tsconfig.base.json

```diff
@@ -16,6 +16,7 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@openmined/courses/*": ["apps/courses/src/*"],
```

Comment:
What does this do? Ideally we should add all types to the shared types library in the `libs` folder, not add them to the app itself.

Reply:
import { MY_TYPE } from '@openmined/shared/types';